understanding/content-on-hover-or-focus.html

```diff
@@ -55,348 +53,279 @@
-                     over the trigger. When the added content is large, magnified views may mean that the
-                     user needs to scroll or pan to completely view it, which is impossible unless the
-                     user is able to move their pointer off the trigger without the additional content
-                     disappearing.
+                  <p>この条件の意図は、ターゲットにホバーすることで現れるかもしれない追加のコンテンツがそれ自身にもホバーできるように保証することである。ホバーで現れるコンテンツは、利用者がマウスポインタをトリガーの上に固定することを求められた場合、知覚するのが難しい又は不可能な可能性がある。追加されたコンテンツが大きい場合、拡大されたビューは、利用者がそのコンテンツを完全に表示するためにスクロール又はパンする必要があることを意味し、これは、追加コンテンツが消えないように利用者がトリガーからポインタを移動できない限り不可能である。
                   </p>
                   			
-                  <p>Another common situation is when large pointers have been selected via platform settings
-                     or assistive technology. Here, the pointer can obscure a significant area of the additional
-                     content. A technique to view the content fully in both situations is to move the mouse
-                     pointer directly from the trigger onto the new content.  This capability also offers
-                     significant advantages for users who utilize screen reader feedback on mouse interactions.
-                     This condition generally implies that the additional content overlaps or is positioned
-                     adjacent to the target.
+                  <p>もう一つの一般的な状況は、大きいポインタがプラットフォームの設定又は支援技術により選択されていた場合である。ここでは、ポインタが追加コンテンツの重要な領域を隠してしまう可能性がある。両方の状況でコンテンツを完全に見る方法は、マウスポインタをトリガーから新しいコンテンツに直接移動させることである。この機能はまた、マウスとのインタラクションのフィードバックにスクリーンリーダーを使用する利用者に重要な利点を提供する。この条件は、ほとんどの場合、追加コンテンツがターゲットに重なる又はそれが隣接して配置されていることを暗示する。
+
+
                   </p>
                   			
                </section>
                <section id="persistent">
                   			
-                  <h3>Persistent</h3>
+                  <h3>表示が継続される</h3>
                   			
-                  <p>The intent of this condition is to ensure users have adequate time to perceive the
-                     additional content after it becomes visible.  Users with disabilities may require
-                     more time for many reasons, such as to change magnification, move the pointer, or
-                     simply to bring the new content into their visual field.  Once it appears, the content
-                     should remain visible until:
+                  <p>この条件の意図は、利用者に追加コンテンツが可視になった後、知覚するための十分な時間があるようにすることである。障害のある利用者は拡大率を変更する、ポインタを動かす、又は可視領域に新しいコンテンツを持ってくるなどの様々な理由により、より多くの時間が必要かもしれない。一度現れたら、次のことをする時までコンテンツは可視のままにするべきである:
+
+
                   </p>
                   			
                   <ul>
                      			
-                     <li>The user removes hover or focus from the trigger and the additional content, consistent
-                        with the typical user experience;
+                     <li>利用者が、典型的なユーザ体験と一致して、トリガー及び追加コンテンツからホバー又はフォーカスを取り除く。
```

Comment:
「典型的なユーザ体験と一致して」が条件になっているように読めますが、そうではなく、removes hover or focus によってコンテンツが消えるのは一般的な体験であるという話ですね。こんな感じにしてはどうでしょうか。

```suggestion
                     <li>利用者がトリガー及び追加コンテンツからホバー又はフォーカスを取り除く。これは典型的なユーザ体験と一致するものである。
```